From 27b5a37a371a89a1e83baf6b444f2b34c0873294 Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew.bayer@gmail.com>
Date: Fri, 18 Mar 2011 07:52:53 -0700
Subject: [PATCH 21/51] CLOUDERA-BUILD. Making sure Hive looks for Hadoop tarball in local Maven repo first.

---
 cloudera/do-release-build |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/cloudera/do-release-build b/cloudera/do-release-build
index ad79722..6a4d6bd 100755
--- a/cloudera/do-release-build
+++ b/cloudera/do-release-build
@@ -14,6 +14,7 @@ cd $RELEASE_DIR
 mkdir -p build/.ivy
 ant $EXTRA_ANT_FLAGS \
   -propertyfile cloudera/build.properties \
+  -Dreactor.repo=file://${HOME}/.m2/repository \
   tar
 
 # Call Maven build for install/deploy of artifacts
-- 
1.7.0.4

