From 31775516718f7b4021b5692a6fa9198a3fb35a64 Mon Sep 17 00:00:00 2001
From: Roman Shaposhnik <rvs@cloudera.com>
Date: Fri, 27 May 2011 14:56:27 -0700
Subject: [PATCH 30/51] DISTRO-150. Should /var/lib/hive/metastore/scripts/* be in /usr/lib/hive instead?

---
 cloudera/install_hive.sh |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/cloudera/install_hive.sh b/cloudera/install_hive.sh
index b90edb1..c93b0c6 100755
--- a/cloudera/install_hive.sh
+++ b/cloudera/install_hive.sh
@@ -143,5 +143,3 @@ chmod 755 ${PYTHON_DIR}/hive_metastore/*-remote
 
 # Dir for Metastore DB
 install -d -m 1777 $PREFIX/var/lib/hive/metastore/
-mkdir $PREFIX/var/lib/hive/metastore/scripts
-cp -R ${BUILD_DIR}/scripts/metastore/upgrade $PREFIX/var/lib/hive/metastore/scripts
-- 
1.7.0.4

