From 924f76e5adcd10503ef72062e17807da3e4091c3 Mon Sep 17 00:00:00 2001
From: carl <carl@cloudera.com>
Date: Thu, 23 Jun 2011 16:53:16 -0700
Subject: [PATCH 33/51] CLOUDERA-BUILD. Set Hive version to 0.7.1-cdh3u1-SNAPSHOT

---
 build.properties |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/build.properties b/build.properties
index 514c2f9..e3ea471 100644
--- a/build.properties
+++ b/build.properties
@@ -1,6 +1,6 @@
 Name=Hive
 name=hive
-version=0.7.1
+version=0.7.1-cdh3u1-SNAPSHOT
 year=2011
 
 javac.debug=on
-- 
1.7.0.4

